test/extended/authorization/scc.go

```diff
@@ -164,3 +164,56 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
 		o.Expect(pod.Annotations[securityv1.ValidatedSCCAnnotation]).To(o.Equal("restricted-v2"))
 	})
 })
+
+var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLIWithPodSecurityLevel("required-scc", psapi.LevelPrivileged)
+
+	g.It("Workloads fail with incorrect/non-existent for required-scc annotation", func() {
+
```

Comment:
Per https://github.com/openshift/enhancements/blob/master/enhancements/authentication/custom-scc-preemption-prevention.md "If the requested SCC does not exist in the system, admission fails with an error" and "If the SCC is applicable to the workload, it gets/is used, else admission fails with an error", it is the admission that fails instead of the workload. So, to be precise:
```suggestion
	g.It("SCC admission fails for incorrect/non-existent required-scc annotation", func() {
```
Also, blank line immediately after `g.It` is usually not needed.